index.js

```diff
@@ -1,41 +1,244 @@
+            ${title2[1]}
+          </h3>
+          <p class="card_description">
+            ${description2}
+          </p>
+          <ul class="tech_stack">
+            <li class="tech">${html}</li>
+            <li class="tech">${bootstrap}</li>
+            <li class="tech">${ruby}</li>
+          </ul>
+          <button class="card_btn bttn" type="button">${btn}</button>
+          </div>
+          <div class="card card_4" >
+
+          <h3 class="card_title">
+            ${title2[0]}<br />
+            ${title2[1]}
+          </h3>
+          <p class="card_description">
+            ${description2}
+          </p>
+          <ul class="tech_stack">
+            <li class="tech">${html}</li>
+            <li class="tech">${bootstrap}</li>
+            <li class="tech">${ruby}</li>
+          </ul>
+          <button class="card_btn bttn" type="button">${btn}</button>
+        </div>
+        <div class="card card_5" >
+
+          <h3 class="card_title">
+            ${title2[0]}<br />
+            ${title2[1]}
+          </h3>
+          <p class="card_description">
+            ${description2}
+          </p>
+          <ul class="tech_stack">
+            <li class="tech">${html}</li>
+            <li class="tech">${bootstrap}</li>
+            <li class="tech">${ruby}</li>
+          </ul>
+          <button class="card_btn bttn" type="button">${btn}</button>
+        </div>
+        <div class="card card_6" >
+
+          <h3 class="card_title">
+            ${title2[0]}<br />
+            ${title2[1]}
+          </h3>
+          <p class="card_description">
+            ${description2}
+          </p>
+          <ul class="tech_stack">
+            <li class="tech">${html}</li>
+            <li class="tech">${bootstrap}</li>
+            <li class="tech">${ruby}</li>
+          </ul>
+          <button class="card_btn bttn" type="button">${btn}</button>
+          </div>
+          `;
+});
+
+portfolio.appendChild(cardsDiv);
+
+const popup = makeDiv();
+popup.className = 'modal';
+popup.innerHTML = modalData.map((item) => {
+  const {
+    title, technologies, images, text, text2, buttons,
+  } = item;
+  const [title1, title2] = title;
+  const [html, bootstrap, ruby] = technologies;
+  const [img1, img2, icon] = images;
+  const [btn1, btn2] = buttons;
+  return ` <div class="heading">
+          <div class="pop-title">
+            <h3 class="modal_ttl title1">
+              ${title2}
+            </h3>
+            <h3 class="modal_ttl title2">${title1}</h3>
+            <i class="fa-regular fa-x cls-modal"></i>
+          </div>
+          <ul class="modal_tech">
+            <li class="md_tech">${html}</li>
+            <li class="md_tech">${bootstrap}</li>
+            <li class="md_tech">${ruby}</li>
+          </ul>
+        </div>
+        <div class="content">
+          <div class="image">
+            <img
+              class="pop_desk"
+              src=${img1}
+              alt="portfolio"
+            />
```

Comment:
**1-** In the figma design, the popup starts from the top of the page but in your case, it is under the navbar please consider making it to the top of the page. 

**2-** **Moreover**, in design, the home page is blurred when the popup opens so consider resolving it also to meet the design guidelines. And remember this blue effect needs also to implemented in the desktop version also ✅ 

## CONFUSION AND SOLUTION:-

Actually, the design you see in Figma is what you see on mobile means it has the mobile top where it shows battery health which obviously is not part of the design so when designing consider focusing on the design under it.

![image](https://user-images.githubusercontent.com/94223367/233417607-091fe187-b7ec-4fa5-bea4-4e1af6feaae9.png)

**3-** Currently, the desktop version is also not in accordance with the design of the template buttons are overflowing and the modal is not centered with respect to the screen.

![image](https://user-images.githubusercontent.com/94223367/233419222-ae4e92cd-017f-4a45-bdaf-a1ae484ee4fa.png)


![image](https://user-images.githubusercontent.com/94223367/233419205-56eb102f-a436-4421-9b9d-4fa879f7dbd0.png)